.github/workflows/early_final_release_checker.yml

```diff
@@ -107,14 +107,12 @@ jobs:
     needs:
       - archive-run-small
       - archive-notify
-    env:
-      OUTPUT1: ${{needs.archive-notify.outputs.output1}}
-      # If Slack message includes new data, create a Github issue
-    if: ${{ always() && (contains(OUTPUT1, "CREATE") || contains(OUTPUT1, "UPDATE") || inputs.create_github_issue == true) }}
     steps:
       - uses: actions/checkout@v4
       - name: Create an issue
         uses: JasonEtco/create-an-issue@v2.9.2
+        # If Slack message includes new data, create a Github issue
+        if: ${{ always() && ( inputs.create_github_issue == true || (contains(needs.archive-notify.outputs.output1, 'CREATE') || contains(needs.archive-notify.outputs.output1, 'UPDATE'))) }}
```

Comment:
Not blocking and I suspect this was cut-and-pasted from elsewhere but why is the `always()` necessary in this condition?

Reply:
It's to address a very unintuitive behavior of GHA. See https://github.com/actions/runner/issues/491